mautic/mautic
# Disable merge identified contacts by tracking code

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR added new option to configuration to disable merge identified contacts from tracking code. This is helpful for shared computers. This allow switch contact/or create new one if not same like current tracked contact.


![image](https://user-images.githubusercontent.com/462477/61954579-692edc00-afb9-11e9-9c29-eb72cbffd459.png)


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Enable new option in configuration > Trackign settings > Disable merge identified contacts 
3. Enable Publicly updatable for unique custom fields (email propably)
4. Create page with tracking code and use

`mt('send', 'pageview', {email: 'test@gmail.com'});`

See test@gmail.com is created and should be tracked contact now (check mtc_id cookie)

5. Change tracking code to another email address:

`mt('send', 'pageview', {email: 'test2@gmail.com'});`

See test2@gmail.com is created should be tracked contact now (check changed mtc_id cookie)

6. Change tracking code to email address from step 4:

`mt('send', 'pageview', {email: 'test@gmail.com'});`

See test@gmail.com is updated and should be tracked contact now (check mtc_id cookie)
